docker-hub/download-rate-limit.md

```diff
@@ -77,12 +78,19 @@ Which should return headers including these:
 ```http
 ratelimit-limit: 100;w=21600
 ratelimit-remaining: 76;w=21600
+docker-RateLimit-Source: 0.0.0.0
 ```
 
-This means my limit is 100 per 21600 seconds (6 hours), and I have 76 pulls remaining.
+This means my limit is 100 per 21600 seconds (6 hours), I have 76 pulls remaining and I am limited on the ip 0.0.0.0 (not a valid ip address, used as an example).
 
 > Remember that these headers are best-effort and there will be small variations.
 
+### docker-RateLimit-Source
+
+This header will show what it used to limit you. It can help diagnose any issue.
+If you are not authenticated, it will show your ip address.
+If you are authenticated, it will show your account name.
```

Comment:
We actually show the account ID. So for one of my users on the Personal plan I received these relevant headers
```
...
ratelimit-limit: 200;w=21600
ratelimit-remaining: 200;w=21600
docker-ratelimit-source: 186e63fa-abf0-4deb-a711-0dbd34c4a821
...
```

Which sort of matches what is returned from a users endpoint
```
🚀 curl -sSL https://hub.docker.com/v2/users/ingshtrom
{"id": "186e63faabf04deba7110dbd34c4a821", "username": "ingshtrom", "full_name": "Alex Hokanson", "location": "", "company": "", "profile_url": "https://github.com/ingshtrom", "date_joined": "2015-09-24T13:29:22.607692Z", "gravatar_url": "https://secure.gravatar.com/avatar/3717121f7d7aad78bc590cdb8cc9196a.jpg?s=80&r=g&d=mm", "type": "User"}%
```

@RomainBelorgey I don't think there is anything we can do in the rate limiting system to make this easier for users without significant changes. Maybe add some documentation explaining that this is the user ID and you can compare it against what the `/v2/users/<namespace>` returns?

Reply:
Thanks, I should have check that better (only did anonymous checks).
I can add more information on how to check the account from the id.